python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/defs_bgp_additional_paths.schema.yml

```diff
@@ -17,10 +18,9 @@ $defs:
           ecmp: All paths in best path ECMP group.
           limit: Limit to n eligible paths.
           disabled: Disable sending any path.
-        valid_values: [ 'any', 'backup', 'ecmp', 'limit', 'disabled' ]
```

Comment:
This should not be removed.

Reply:
don't think I've deleted that.. else it was by accident